dev_requirements/linter-requirements.txt

```diff
@@ -0,0 +1,14 @@
+bandit==1.7.0
```

Comment:
This is a nitpick, but can we consistently use `_` or `-` for file and directory names?

Reply:
In this repo, there is a mostly consistent policy that directories use `_` and files use `-`.

I do not see the value in this policy. 
I do not know how this came to be.
I am not going suggest we enforce it in any way. 
But since this already meets the policy, I suggest we leave it as is.